app/templates/index.hbs

```diff
@@ -27,11 +27,13 @@
       {{/each}}
     </div>
     <div class="ui hidden divider"></div>
-    <div class="ui centered grid">
-      <div class="row">
-        {{#link-to "explore" (query-params cfs='open') class='ui blue button'}}{{t 'Show more calls for speakers'}}{{/link-to}}
+    {{#if (gt callForSpeakersEvents 3)}}
```

Comment:
It's not the correct way `callForSpeakersEvents` returns the queried events not there number. Also, the comparison works here due to the `Ember Arith Helper`. I think a new property `CFSeventsnumber` should be used for comparison, I am also not sure if adding new property will be optimal or not. Hence, I will wait for the review of other mentors.

Reply:
@kushthedude  i understand what you are trying to say . So i changed the comparison here from  {{#if (gt callForSpeakersEvents 3)}} to  {{#if (gt callForSpeakersEvents.length 3)}}
now  here is no reason to say it false in my opinion .

Reply:
Please check it .

Reply:
@uds5501 can you please check it again that it is also working fine ?